framework/src/main/java/org/checkerframework/framework/type/TypeFromMemberVisitor.java

```diff
@@ -58,7 +72,8 @@ public AnnotatedTypeMirror visitMethod(MethodTree node, AnnotatedTypeFactory f)
 
         // TODO: Needed to visit parameter types, etc.
         // It would be nicer if this didn't decode the information from the Element and
-        // instead also used the Tree.
+        // instead also used the Tree. If this is implemented, then care needs to be taken to put
+        // any alias declaration annotations in the correct place for return types that are arrays.
```

Comment:
A reference to `ElementAnnotationUtil.addDeclarationAnnotationsFromElement` might be good here.

Reply:
Done below.